### DISCUSSION TITLE
Update to pydantic >= 2.0.0

### DISCUSSION THREAD

**Discussion by isaacn-lab** [datapane/datapane#394] [General]
Hello,

Currently `pydantic` is locked to version less than 2.0.0 (`pydantic = "^1.6.0"`). Do you plan to update it in the near future?
Thank you in advance!

-Nikita

**Comment by Samreay** [comment]
An update to pydantic would be amazing, even if its just to allow pydantic 2 and use the `.v1` API

**Comment by mands** [comment]
resolved in final 0.17.0 release

**Reply by isaacn-lab** [reply]
thank you for the update!
